cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,48 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Correctness Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <ul class="list-fields list-input" role="group" aria-labelledby="show_correctness_label">
```

Comment:
Much nicer without all that unneeded HTML.. cf https://github.com/edx/edx-platform/pull/14737/commits/541b3c9d4d39e3d84910ea242c86c9dc4569e069#diff-f60a9f957169beadefa5fedf9607dfeaR4